en/docs/chapter_array_and_linkedlist/linked_list.md

```diff
@@ -404,51 +404,51 @@ Constructing a linked list is a two-step process: first, initializing each node
 
-### Inserting a Node
+### Inserting a node
 
 Inserting a node into a linked list is very easy. As shown in the figure, let's assume we aim to insert a new node `P` between two adjacent nodes `n0` and `n1`. **This can be achieved by simply modifying two node references (pointers)**, with a time complexity of $O(1)$.
 
 By comparison, inserting an element into an array has a time complexity of $O(n)$, which becomes less efficient when dealing with large data volumes.
 
-![Linked List Node Insertion Example](linked_list.assets/linkedlist_insert_node.png)
+![Linked list node insertion example](linked_list.assets/linkedlist_insert_node.png)
 
 ```src
 [file]{linked_list}-[class]{}-[func]{insert}
 ```
 
-### Deleting a Node
+### Deleting a node
 
 As shown in the figure, deleting a node from a linked list is also very easy, **involving only the modification of a single node's reference (pointer)**.
 
 It's important to note that even though node `P` continues to point to `n1` after being deleted, it becomes inaccessible during linked list traversal. This effectively means that `P` is no longer a part of the linked list.
 
-![Linked List Node Deletion](linked_list.assets/linkedlist_remove_node.png)
+![Linked list node deletion](linked_list.assets/linkedlist_remove_node.png)
 
 ```src
 [file]{linked_list}-[class]{}-[func]{remove}
 ```
 
-### Accessing Nodes
+### Accessing nodes
 
 **Accessing nodes in a linked list is less efficient**. As previously mentioned, any element in an array can be accessed in $O(1)$ time. In contrast, with a linked list, the program involves starting from the head node and sequentially traversing through the nodes until the desired node is found. In other words, to access the $i$-th node in a linked list, the program must iterate through $i - 1$ nodes, resulting in a time complexity of $O(n)$.
 
 ```src
 [file]{linked_list}-[class]{}-[func]{access}
 ```
 
-### Finding Nodes
+### Finding nodes
 
 Traverse the linked list to locate a node whose value matches `target`, and then output the index of that node within the linked list. This procedure is also an example of linear search. The corresponding code is provided below:
 
 ```src
 [file]{linked_list}-[class]{}-[func]{find}
 ```
 
-## Arrays vs. Linked Lists
+## Arrays vs. linked lists
 
 The table below summarizes the characteristics of arrays and linked lists, and it also compares their efficiencies in various operations. Because they utilize opposing storage strategies, their respective properties and operational efficiencies exhibit distinct contrasts.
 
-<p align="center"> Table <id> &nbsp; Efficiency Comparison of Arrays and Linked Lists </p>
+<p align="center"> Table <id> &nbsp; Efficiency comparison of arrays and lnked lists </p>
```

Comment:
typo: Efficiency comparison of arrays and _lnked_ lists -> Efficiency comparison of arrays and _linked_ lists